relations.csv

```diff
@@ -4067,3 +4067,80 @@ HGNC,MIRLET7A3,isa,FPLX,MIRLET7A
 FPLX,MIRLET7F,isa,FPLX,MIRLET7
 HGNC,MIRLET7F1,isa,FPLX,MIRLET7F
 HGNC,MIRLET7F2,isa,FPLX,MIRLET7F
+HGNC,HTR1A,isa,FPLX,5-HT
+HGNC,HTR1B,isa,FPLX,5-HT
+HGNC,HTR1D,isa,FPLX,5-HT
+HGNC,HTR1E,isa,FPLX,5-HT
+HGNC,HTR1F,isa,FPLX,5-HT
+HGNC,HTR2A,isa,FPLX,5-HT

```

Comment:
could you make more fine granular families like HTR1, HTR2, HTR3, HTR4, and HTR5 that go in the middle?